### PR TITLE
KOGITO-7937: Dashbuilder Editor Viewer

### PR DESCRIPTION
**Pull request by jesuino**
I am adding an envelope to view YAML/JSON dashboards definitions. This envelope will be the base for VSCode extension and  the definitive way to render Dashbuilder using Multiplying platform.

### CODE REVIEW

**Comment by jesuino** [comment]
The most convenient way to try this project is by running `pnpm start` on `dashbuilder-viewer`.

File: packages/dashbuilder-viewer/dev-webapp/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            from: path.resolve(__dirname, "../dist/dashbuilder-client"),
+            to: "./",
+          },
+          {
+            from: path.resolve(__dirname, "./static/envelope/dashbuilder-viewer-envelope.html"),
+            to: "./dashbuilder-viewer-envelope.html",
+          },
+        ],
+      }),
+      new MonacoWebpackPlugin({
```
Comment (caponetto) [inline packages/dashbuilder-viewer/dev-webapp/webpack.config.js]: As there is no textual part, is this necessary?

File: packages/dashbuilder-viewer/package.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    "@kie-tools-core/editor": "workspace:*",
+    "@kie-tools-core/guided-tour": "workspace:*",
+    "@kie-tools-core/notifications": "workspace:*",
+    "@kie-tools-core/operating-system": "workspace:*",
+    "@kie-tools-core/patternfly-base": "workspace:*",
+    "@kie-tools-core/workspace": "workspace:*",
+    "@patternfly/react-core": "^4.157.3",
+    "@patternfly/react-icons": "^4.11.17",
+    "monaco-editor": "^0.33.0",
+    "monaco-yaml": "^4.0.0",
```
Comment (caponetto) [inline packages/dashbuilder-viewer/package.json]: As there is no textual part, are these necessary?

File: packages/dashbuilder-viewer/package.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    "@babel/preset-react": "^7.16.0",
+    "@kie-tools-core/webpack-base": "workspace:*",
+    "@kie-tools/dashbuilder-client": "workspace:*",
+    "@kie-tools/eslint": "workspace:*",
+    "@kie-tools/root-env": "workspace:*",
+    "@kie-tools/tsconfig": "workspace:*",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@types/jest": "^26.0.23",
+    "@types/jest-when": "^2.7.4",
+    "@types/react": "^17.0.6",
```
Comment (caponetto) [inline packages/dashbuilder-viewer/package.json:59]:
```suggestion
    "@types/react": "^17.0.6",
    "@types/react-dom": "^17.0.5",
```
FYI I needed to add the dep `"@types/react-dom"` to be able to run `pnpm start`.

**Comment by jesuino** [comment]
thanks @caponetto  - I made the changes!

**Review by caponetto — APPROVED** [review]

File: packages/dashbuilder-viewer/dev-webapp/static/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE html>
+<html lang="en" class="pf-m-redhat-font">
+  <head>
+    <title>Dashbuilder Editor</title>
```
Comment (barboras7) [inline packages/dashbuilder-viewer/dev-webapp/static/index.html]: Should be this Dashbuilder Editor or Dashbuilder Viewer?

**Review by barboras7 — COMMENTED** [review]
Hi @jesuino!

The PR looks good. Just some minor comment inline. Also if it's important (to be honest, I don't know if it's an issue :smile: ), please, fix the year in the license for some files (you have 2019, 2020 and 2021 there). 

Also, what should be the way to send some content to Dashbuilder viewer? 

Thanks

**Comment by jesuino** [comment]
Hello @barboras7 

Thanks for your review, really appreciate! I made the changes in license headers.

>> Also, what should be the way to send some content to Dashbuilder viewer?

You can edit `TEST_CONTENT` in `App.tsx` while running the project in dev mode.

It will be used to visualize data coming from other platforms (e.g. VSCode and Chrome Extension). The next PR will be the VSCode extension that will make this clear!

**Review by barboras7 — APPROVED** [review]
Looks good, thanks @jesuino!

**Review by ederign — APPROVED** [review]

**Pull request merged**